projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -1,39 +1,70 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
 use Automattic\Jetpack\Assets;
 use Automattic\Jetpack\Status;
-/*
-Plugin Name: Jetpack Carousel
-Plugin URL: https://wordpress.com/
-Description: Transform your standard image galleries into an immersive full-screen experience.
-Version: 0.1
-Author: Automattic
-
-Released under the GPL v.2 license.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-*/
-class Jetpack_Carousel {
 
+/**
+ * Module: Jetpack Carousel
+ *
+ * @package automattic/jetpack
+ */
```

Comment:
Ah, thanks!

Reply:
Does the suggested change include `class Jetpack_Carousel {`? I can manually commit otherwise.

Reply:
Unfortunately GitHub's UI gets a bit confused if you try to do a suggestion on a range that includes deleted lines. To tell the truth I'm not sure what exactly it would to if you try to use GitHub's UI to apply it.